tutorials/relative_time_constraints/index.md

```diff
@@ -0,0 +1,629 @@
+---
+title:  Dating with relative constraints
+subtitle: Constraining the relative order of nodes when dating a phylogeny
+authors: Dominik Schrempf, Gergely Szöllősi, and Bastien Boussau
+level: 3
+prerequisites:
+- intro
+- mcmc
+- ctmc
+index: true
+---
+
+Introduction
+--
+{:.section}
+
+In phylogenetics, the term __dating__ denotes the inference of node ages of
+phylogenies. For example, dating a species tree involves inference of the times
+when the species split. Dating phylogenies is central to understanding the
+evolution of life on Earth. However, evolutionary sequences evolve with
+different rates, an observation that has been termed __relaxed molecular
+clock__. In general, it is difficult to map a phylogeny obtained from a multiple
+sequence alignment, and with branch lengths measured in average number of
+substitutions per site, to a phylogeny with branch lengths measured in actual
+units of time. For this reason, the molecular clocks are calibrated with
+information gained from __fossils__, which can be accurately dated.
+Incorporating fossil information enables __calibration__ of node ages of a
+phylogeny.
+
+In this tutorial, we explore another possibility to improve on the accuracy of
+dating a phylogeny. Namely, horizontal gene transfers and ancient evolutionary
+relationships such as symbioses are informative about the order of nodes of a
+phylogeny of interest. That is, __horizontal gene transfers__ define __relative
+node constraints__. Relative node constraints can be particularly helpful when
+the geological record is sparse, for example, for microorganisms, which
+represent the vast majority of extant and extinct biodiversity. Combination of
+node calibrations with relative node constraints can significantly improve both
+accuracy and resolution of molecular clock estimates {% cite Szollosi2020 %}.
+
+Definitions
+-------------
+{:.section}
+
+__Alignment__: Multiple sequence alignment.
+
+__Timetree__: Phylogeny with branch lengths measured in units of time. If the
+leaves have been sampled at the same time, for example, at the present, a
+time-tree is ultrametric.
+
+__Branch-length tree__: Phylogeny with branch lengths measured in expected
+numbers of substitutions. Usually, branch-length trees are obtained from a
+phylogenetic analysis of an alignment.
+
+__Calibration__: Absolute node calibration; an estimate of the age of a node of
+a timetree. This is usually associated with a prior describing the uncertainty
+associated to this node age.
+
+__Constraint__: Relative node order constraint, which specifies the relative
+order in time of two nodes of a tree (e.g. node A is older than node B).
+
+Getting Started
+------------------
+{:.section}
+
+This tutorial was tested against the development branch,
+available at [commit 06c1cac](https://github.com/revbayes/revbayes/commit/06c1cac4b9e62185cd8db45f638f152e30045ab8).
```

Comment:
I think its fine to mention this. I would assume this will be removed in the future at some point, but it might be nice to have there for troubleshooting purposes.

Reply:
I think it is good to have the commit. It can be removed anytime!